src/Xamarin.Android.Build.Tasks/Xamarin.Android.Build.Tasks.targets

```diff
@@ -405,4 +407,12 @@
   <Target Name="_Remove32BitLinux" Condition="Exists ('$(OutputPath)lib32\libZipSharpNative.so')" AfterTargets="CopyFilesToOutputDirectory">
     <Delete Files="$(OutputPath)lib32\libZipSharpNative.so" />
   </Target>
+
+  <Target Name="_SetAssemblyVersion"
+      DependsOnTargets="GetXAVersionInfo"
+      BeforeTargets="GetAssemblyAttributes">
```

Comment:
Great, the build seems green!

But the version didn't change:

![image](https://user-images.githubusercontent.com/840039/128563758-ad569463-b20b-4e23-84a6-5cb5e606cf3a.png)

I think we just need to make sure our target runs before `GetAssemblyVersion`:

![image](https://user-images.githubusercontent.com/840039/128563697-0a074bf6-59f0-4aab-b8b6-5f13afee105e.png)

Such as:

```suggestion
  <Target Name="_SetAssemblyVersion"
      DependsOnTargets="GetXAVersionInfo"
      BeforeTargets="GetAssemblyVersion">
```